### PR TITLE
Lamba Layer Release - March 2022 Update layer versions in README.md

### PR DESCRIPTION
**Pull request by vasireddy99**
Description:

This PR updates the README.md and changelog.md with respective versions for Python, NodeJS, Java-agent, Java-wrapper and collector layers

Documentation:
It is a documentation change

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## 2022-03-07 Release
+**AWS Distro For OpenTelemetry lambda now supports ARM64 Architecture**
+- Python3.8 layer **aws-otel-python38-ver-1-9-1** contains OpenTelemetry Python `v1.9.1` with the AWS Python Extension `v2.0.1`
+- Nodejs layer [aws-otel-nodejs-ver-1-0-1](https://aws-otel.github.io/docs/getting-started/lambda/lambda-js) contains OpenTelemetry JavaScript Core `v1.0.1` with AWS Lambda Instrumentation `v0.29.0`
+- Java-Wrapper layer **aws-otel-java-wrapper-ver-1-11-1** contains OpenTelemetry Java `v1.11.1`
+- Java-Agent layer **aws-otel-java-agent-ver-1-11-1** contains AWS OpenTelemetry Java instrumentation `v1.11.1`
+- Collector layer **aws-otel-collector-ver-0-45-0** contains ADOT Collector for Lambda `v0.17.0`.
```
Comment (bryan-aguilar) [inline CHANGELOG.md:10]: Is ADOT Collector for Lambda the correct terminology?

File: README.md [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
-- Collector layer **aws-otel-collector-ver-0-43-1** contains ADOT Collector for Lambda `v0.16.0`. Compatible with [.NET](https://aws-otel.github.io/docs/getting-started/lambda/lambda-dotnet) and [Go](https://aws-otel.github.io/docs/getting-started/lambda/lambda-go) runtimes.
+- Nodejs layer [aws-otel-nodejs-ver-1-0-1](https://aws-otel.github.io/docs/getting-started/lambda/lambda-js) contains OpenTelemetry JavaScript Core `v1.0.1` with AWS Lambda Instrumentation `v0.29.0`
+- Java-Wrapper layer [aws-otel-java-wrapper-ver-1-11-1](https://aws-otel.github.io/docs/getting-started/lambda/lambda-java) contains OpenTelemetry Java `v1.11.1`
+- Java-Agent layer [aws-otel-java-agent-ver-1-11-1](https://aws-otel.github.io/docs/getting-started/lambda/lambda-java-auto-instr) contains AWS OpenTelemetry Java instrumentation `v1.11.1`
+- Collector layer **aws-otel-collector-ver-0-45-0** contains ADOT Collector for Lambda `v0.17.0`. Compatible with [.NET](https://aws-otel.github.io/docs/getting-started/lambda/lambda-dotnet) and [Go](https://aws-otel.github.io/docs/getting-started/lambda/lambda-go) runtimes.
```
Comment (bryan-aguilar) [inline README.md:8]: Same issue with the term `ADOT Collector for Lambda`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,16 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## 2022-03-07 Release
+**AWS Distro For OpenTelemetry lambda now supports ARM64 Architecture**
```
Comment (NathanielRN) [inline CHANGELOG.md:5]:
```suggestion
**AWS Distro For OpenTelemetry Lambda now supports ARM64 Architecture**
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,16 @@
...
+- Python3.8 layer **aws-otel-python38-ver-1-9-1** contains OpenTelemetry Python `v1.9.1` with the AWS Python Extension `v2.0.1`
+- Nodejs layer [aws-otel-nodejs-ver-1-0-1](https://aws-otel.github.io/docs/getting-started/lambda/lambda-js) contains OpenTelemetry JavaScript Core `v1.0.1` with AWS Lambda Instrumentation `v0.29.0`
+- Java-Wrapper layer **aws-otel-java-wrapper-ver-1-11-1** contains OpenTelemetry Java `v1.11.1`
+- Java-Agent layer **aws-otel-java-agent-ver-1-11-1** contains AWS OpenTelemetry Java instrumentation `v1.11.1`
+- Collector layer **aws-otel-collector-ver-0-45-0** contains ADOT Collector for Lambda `v0.17.0`.
+- Layers cover 16 AWS Regions: us-east-1, us-east-2, us-west-1, us-west-2, ap-south-1, ap-northeast-2, ap-southeast-1, ap-southeast-2, ap-northeast-1, ca-central-1, eu-central-1, eu-west-1, eu-west-2, eu-west-3, eu-north-1, sa-east-1
```
Comment (NathanielRN) [inline CHANGELOG.md:11]:
```suggestion
- x86_64 Layers cover 16 AWS Regions: us-east-1, us-east-2, us-west-1, us-west-2, ap-south-1, ap-northeast-2, ap-southeast-1, ap-southeast-2, ap-northeast-1, ca-central-1, eu-central-1, eu-west-1, eu-west-2, eu-west-3, eu-north-1, sa-east-1
- ARM64 Layers cover 10 AWS Regions: us-east-1, us-east-2, us-west-2, ap-south-1, ap-southeast-1, ap-southeast-2, ap-northeast-1, eu-central-1, eu-west-1, eu-west-2
```

**Review by NathanielRN — APPROVED** [review]
Left some wording suggestions. Thanks!

File: README.md (hunk 2 above)
Comment (NathanielRN) [inline README.md:8]: Need to add `<amd64|arm64>-` to all of these lines.

**Review by NathanielRN — CHANGES_REQUESTED** [review]
Critical changes to the architecture component of the layer names.

**Comment by vasireddy99** [comment]
> Critical changes to the architecture component of the layer names.

Thanks for the call out, committed those changes

**Review by NathanielRN — APPROVED** [review]

**Pull request merged**